### PR TITLE
Release/2.1.0

### PR DESCRIPTION
**Pull request by markmpeterson**
Updated release files for version 2.1.0.

### CODE REVIEW

File: docker-compose/README.md [hunk 1]
```diff
@@ -13,84 +11,101 @@ These files provide example docker-compose files for setting up and configuring
...
 
 
-# Resources for Core and Edge Facilities' Hosts
-Production and production-like deployment toplogies generally follow a "hub and spoke" pattern where certain services can be grouped in the "hub" referred to here as `core`; other services on the "spoke", which we referred to as `edge`.
+# Resources for Control and Edge Hosts
+Production and production-like deployment toplogies generally follow a "hub and spoke" pattern where certain services can be grouped in the "hub" referred to here as `control node`; other services on the "spoke", which we referred to as `edge node`.
 
-![Example topology with command line variables.](https://github.com/ns1/ns1-privatedns/blob/release/1.1.0/docker-compose/figure1.PNG?raw=true)
-**Figure 1**. Example topology with `core-compose.yml` and `edge-compose.yml` command line variables for reference.
+![Example topology with command line variables.](https://github.com/nsone/ns1-privatedns/blob/release/2.1.0-usage-info/docker-compose/figure2.PNG?raw=true)
```
Comment (mburtless) [inline docker-compose/README.md]: This link probably won't work as intended.  Image should be uploaded to this repo and link should point to that instead.

File: docker-compose/control-compose.yml [hunk 2]
```diff
@@ -2,11 +2,11 @@
 version: '3.2'
 services:
   data:
-    image: ns1inc/privatedns_data:${TAG:-1.1.1}
+    image: nsone/standalone_data:${TAG:-2.1.0}
```
Comment (mburtless) [inline docker-compose/control-compose.yml]: Are we no longer using the `ns1inc` dockerhub org and `privatedns` containers?  All images in this compose are pointing to `nsone` org/`standalone` containers

File: docker-compose/docker-compose.yml [hunk 3]
```diff
@@ -2,10 +2,10 @@
 version: '3.2'
 services:
   data:
-    image: ns1inc/privatedns_data:${TAG:-1.1.1}
+    image: nsone/standalone_data:${TAG:-2.1.0}
```
Comment (mburtless) [inline docker-compose/docker-compose.yml]: All images in this compose are pointing to `nsone` org/`standalone` containers

File: docker-compose/edge-compose.yml [hunk 4]
```diff
@@ -2,15 +2,21 @@
 version: '3.2'
 services:
   dns:
-    image: ns1inc/privatedns_dns:${TAG:-1.1.1}
+    image: nsone/standalone_dns:${TAG:-2.1.0}
```
Comment (mburtless) [inline docker-compose/edge-compose.yml]: All images in this compose are pointing to `nsone` org/`standalone` containers

File: docker-compose/min-docker-compose.yml [hunk 5]
```diff
@@ -1,13 +1,12 @@
-
+---
 version: '3.2'
 services:
   data:
-    image: ns1inc/privatedns_data:${TAG:-1.1.1}
+    image: nsone/standalone_data:${TAG:-2.1.0}
```
Comment (mburtless) [inline docker-compose/min-docker-compose.yml]: All images in this compose are pointing to `nsone` org/`standalone` containers

File: docker-compose/control-compose.yml (hunk 2 above)
Comment (markmpeterson) [reply docker-compose/control-compose.yml]: Good call.

Comment (markmpeterson) [reply docker-compose/control-compose.yml]: Should've been changed when brought over here.

**Comment by markmpeterson** [comment]
Merging and will create a release next off master

**Pull request merged**